src/Classes/Models/HiringPartnerModel.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: imogen
+ * Date: 27/11/2018
+ * Time: 15:34
+ */
+
+namespace Portal\Models;
+
+
+class HiringPartnerModel
+{
+    private $db;
+
+    /**
+     * HiringPartnerModel constructor.
+     * @param $db
+     */
+    public function __construct(\PDO $db)
+    {
+        $this->db = $db;
+    }
+
+    /**
+     * Gets company size brackets from database
+     * @return array each id and size bracket pair
+     */
+    public function getDropdownData() {
```

Comment:
Bracket should be on next line because it's a method in a class, not a standalone function

Reply:
Fixed in 3274d25